provisioning/roles/mangaki_source/README.md

```diff
@@ -85,6 +85,9 @@ mangaki_source_auth_providers: ['twitter', 'facebook']
 # The path to the web root to store static files in (required).
 mangaki_source_static_root: '/var/www/mangaki/static'
 
+# The path to the web root to store data and snapshots in (required).
+mangaki_source_data_root: '/var/www/mangaki/data'
```

Comment:
I'd prefer `/var/mangaki/data`.

Reply:
"to store X in" => "where X is stored" non ?

Reply:
Done.